easy-ui-react/src/Button/Button.stories.mdx

```diff
@@ -0,0 +1,367 @@
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+Small outlined buttons
+
+<Canvas>
+  <Story
+    name="PrimaryOutlinedSmall"
+    args={{
+      appearance: "outlined",
+      size: "sm",
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+  <Story
+    name="SecondaryOutlinedSmall"
+    args={{
+      color: "secondary",
+      appearance: "outlined",
+      size: "sm",
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+  <Story
+    name="SuccessOutlinedSmall"
+    args={{
+      color: "success",
+      appearance: "outlined",
+      size: "sm",
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+  <Story
+    name="WarningOutlinedSmall"
+    args={{
+      color: "warning",
+      appearance: "outlined",
+      size: "sm",
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+  <Story
+    name="SupportOutlinedSmall"
+    args={{
+      color: "support",
+      appearance: "outlined",
+      size: "sm",
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+  <Story name="InverseOutlinedSmall">
+    <div
+      style={{
+        width: "75px",
+        height: "42px",
+        backgroundColor: "var(--ezui-color-blue-800)",
+        display: "flex",
+        alignItems: "center",
+        justifyContent: "center",
+      }}
+    >
+      <Button color="inverse" appearance="outlined" size="sm" />
+    </div>
+  </Story>
+</Canvas>
+
+With href
+
+<Canvas>
+  <Story
+    name="PrimaryWithHref"
+    args={{
+      href: "https://www.easypost.com/",
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+  <Story
+    name="PrimaryOutlinedWithHref"
+    args={{
+      appearance: "outlined",
+      href: "https://www.easypost.com/",
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+With end icons
+
+<Canvas>
+  <Story name="PrimaryWithEndIcon">
+    <Button endIcon={<Icon symbol={CheckCircleIcon} />} />
+  </Story>
+  <Story name="NeutralWithEndIcon">
+    <Button color="neutral" endIcon={<Icon symbol={AddIcon} />} />
+  </Story>
+</Canvas>
+
+With start icons
+
+<Canvas>
+  <Story name="SuccessWithStartIcon">
+    <Button color="success" startIcon={<Icon symbol={ArrowBackIcon} />} />
+  </Story>
+  <Story name="WarningWithStartIcon">
+    <Button color="warning" startIcon={<Icon symbol={InfoIcon} />} />
+  </Story>
+</Canvas>
+
+Block button
+
+<Canvas>
+  <Story
+    name="PrimaryBlock"
+    args={{
+      isBlock: true,
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+Disabled buttons
+
+<Canvas>
+  <Story
+    name="PrimaryDisabled"
+    args={{
+      isDisabled: true,
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+  <Story
+    name="PrimaryOutlinedDisabled"
+    args={{
+      isDisabled: true,
+      appearance: "outlined",
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+Click events
+
+<Canvas>
+  <Story
+    name="ClickEvent"
+    args={{
+      onPress: () => alert("clicked!"),
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+<ArgsTable story="Default" />
```

Comment:
i tried to write a function that would generate these stories, but i was getting some strange type errors, might revisit this though because it does feel a little heavy to have a bunch of stories manually written out